### ISSUE TITLE
Treeitem issues – tooltip visualized is not read by Jaws

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#898]
Summary

When a “title” attribute is used with role “treeitem”, the tooltip is visualized on hover, but it is not read out by Jaws.

**Example:**[

•	Snippix : https://jsbin.com/lopupuyudu/edit?html,css,js,output)
•	HTML Example (ARIA-APG): 
<div role="tree" aria-label="My tree">
      <div title="tooltip 123" tabindex="0" role="treeitem">Text</div>
</div>

**Workflow:**

Enter tabpanel in examples with TAB
Move away from selected tab and refocus it again

**Expected Result:**
The tooltip visualized on hover should be read by Jaws. issue not reproducible in NVDA

**Additional Information**
JAWS version and build number:Jaws 2025
Operating System and version: Windows 11
Browser and version:Chrome browser

**Comment by JeremiahRogersVispero** [comment]
I cannot reproduce. Using JAWS 2025.2506.170 and Chrome 138.0.7204.169, the title content is being read when the tree receives focus. Please reopen and add additional details, @havvamehfooz, if you believe the issue still exists. Thanks.
